jetty-server/src/main/java/org/eclipse/jetty/server/handler/gzip/GzipHttpOutputInterceptor.java

```diff
@@ -355,7 +354,7 @@ protected Action process() throws Exception
                         int off = slice.arrayOffset() + slice.position();
                         int len = slice.remaining();
                         _crc.update(array, off, len);
-                        _deflater.setInput(array, off, len);  // TODO use ByteBuffer API in Jetty-10
+                        _deflater.setInput(array, off, len);
```

Comment:
@gregw I think its probably best not to use the deflaters ByteBuffer method here, the `CRC32` will need an array from the `ByteBuffer` anyway, and if we do it ourselves we get the use the `ByteBufferPool`.

Reply:
I have added a comment as to why we are not using the ByteBuffer API for deflaters.